packages/amplify-e2e-tests/src/amplify-app-helpers/amplify-app-setup.ts

```diff
@@ -57,14 +54,13 @@ function amplifyAppAngular(projRoot: string, verbose: Boolean = isCI() ? false :
   });
 }
 
-function amplifyAppReact(projRoot: string, verbose: Boolean = isCI() ? false : true) {
+function amplifyAppReact(projRoot: string, verbose: boolean = !isCI()) {
   return new Promise((resolve, reject) => {
-    nexpect
-      .spawn(spawnCommand, { cwd: projRoot, stripColors: true, verbose })
+    spawn(spawnCommand, [], { cwd: projRoot, stripColors: true, verbose })
       .wait('What type of app are you building')
-      .sendline('\r')
+      .sendLine('')
```

Comment:
sendCarriageReturn() could make it more expressive when writing tests and also then sendLine could make the only argument as required and empty passed in argument could be treated as invalid.